fk/main.go

```diff
@@ -65,7 +65,7 @@ Configuration file: %s
 Usage:
 	fk setup
 	fk setup <email>
-	fk secret send <recipient-email-address>
+	fk secret send <recipient-email-address> [--file <filename>]
```

Comment:
I think you meant `[--file=<filename>]`

From the [documentation](http://docopt.org/):

```
Usage:
  naval_fate ship new <name>...
  naval_fate ship <name> move <x> <y> [--speed=<kn>]
  naval_fate ship shoot <x> <y>
  naval_fate mine (set|remove) <x> <y> [--moored|--drifting]
  naval_fate -h | --help
  naval_fate --version
```

We want the equivalent to `[--speed=<kn>]`

More generally though, I found it unnatural specifying the filename *after* the recipient.   I wanted to do:

```
fk secret send /tmp/secret.txt ...
```

Which makes me wonder if it should be:
```
fk secret send <recipient-email-address>                                # stdin version
fk secret send <filename> --to=<recipient-email-address>
```
?